code/gradle/release.gradle

```diff
@@ -215,7 +214,6 @@ task downloadJRE doLast {
                         fileTree(dir: "${projectDir}/jre/${os}/").include("*.${extension}").each { simLib ->
                             copy {
                                 println "Unzip $simLib.name"
-                                def name = simLib.name - '.$extension'
```

Comment:
I thought this was used somewhere else. Maybe it was just an output no longer there.

This should be easy to test. Just delete the JRE folder and call this task.